src/components/inputs/reactHookForm/DirectoryItemsInput.tsx

```diff
@@ -197,11 +218,20 @@ export function DirectoryItemsInput({
                             >
                                 <Chip
                                     size="small"
-                                    sx={{
-                                        backgroundColor:
-                                            item?.specificMetadata?.equipmentType &&
-                                            equipmentColorsMap?.get(item?.specificMetadata?.equipmentType),
-                                    }}
+                                    sx={mergeSx(
+                                        {
+                                            backgroundColor:
+                                                item?.specificMetadata?.equipmentType &&
+                                                equipmentColorsMap?.get(item?.specificMetadata?.equipmentType),
+                                        },
+                                        !watchedElements?.[index]?.[NAME]
```

Comment:
There is just a little flickering pb when adding a new existing filter : 
we see first the filter added with the chip error colors just before having the right colors ....